nautobot_ssot/integrations/infoblox/signals.py

```diff
@@ -156,6 +156,7 @@ def nautobot_database_ready_callback(sender, *, apps, **kwargs):  # pylint: disa
             infoblox_request_timeout = 60
 
         infoblox_sync_filters = _get_sync_filters()
+        infoblox_network_view_to_namespace_map = _get_default_network_view_to_namespace_map()
```

Comment:
I don't understand this, presumably the function is called to allow you to overwrite something, but in the function I don't see anything?

Reply:
This was addressed in #728.